### DISCUSSION TITLE
Get transaction hash

### DISCUSSION THREAD

**Discussion by Thomas-Bibby** [reach-sh/reach-lang#1239] [Q&A] [answered]
Is there anyway I can get the transaction hash from a contract call? I can see the hash in the console with a 404 call but am unsure how to get the hash. Can anyone help? 

![image](https://user-images.githubusercontent.com/62004466/177713986-4c8d37bd-98e1-4ed6-aaa3-70270460bb8c.png)

**Comment by jeapostrophe** [comment, marked as answer]
We do not expose this, primarily because there is not a single transaction hash for a contract call. A contract call may involve many transactions (up to 16+256) and it would not be meaningful to users and there's no programmatic information about it that is available.
